lib/ood_core/job/adapters/kubernetes/k8s_job_info.rb

```diff
@@ -2,8 +2,8 @@
 class OodCore::Job::Adapters::Kubernetes::K8sJobInfo < OodCore::Job::Info
   attr_reader :ood_connection_info
 
-  def initialize(ood_connection_info: {}, **options)
-    super(options)
-    @ood_connection_info = ood_connection_info
+  def initialize(options)
```

Comment:
I assume we're the only consumers of this (hopefully) so changing this interface is OK, if it does indeed change the interface for strictly 2.7 users.